avr/cores/megacommand/MCL/ExtSeqTrack.h

```diff
@@ -70,6 +70,13 @@ struct ext_event_t {
     if (this->micro_timing != that.micro_timing) {
       return this->micro_timing < that.micro_timing;
     }
+    /*
+    // order by lock
```

Comment:
@yatli can you fix this. we want locks before notes.

Reply:
will do.